Resources/Prototypes/Catalog/VendingMachines/Inventories/clothesmate.yml

```diff
@@ -46,6 +46,10 @@
     ClothingUniformJumpsuitHawaiBlue: 2
     ClothingUniformJumpsuitHawaiRed: 2
     ClothingUniformJumpsuitHawaiYellow: 2
+    ClothingUniformJumpskirtBlueDress: 2
+    ClothingUniformJumpskirtYellowDress: 2
+    ClothingUniformJumpskirtBlackDress: 2
+    ClothingUniformJumpskirtRedDress: 2
```

Comment:
There are 10 comments in this file specifically requesting not to add more so please do not ignore it.

Reply:
> There are 10 comments in this file specifically requesting not to add more so please do not ignore it.

Hmm, then how do I solve the problem of getting these dresses in the game? Add them to the theater?